components-core/src/main/java/com/adyen/checkout/components/core/internal/ui/model/SessionParams.kt

```diff
@@ -16,4 +16,5 @@ data class SessionParams(
     val enableStoreDetails: Boolean?,
     val installmentOptions: Map<String, SessionInstallmentOptionsParams?>?,
     val amount: Amount?,
+    val returnUrl: String,
```

Comment:
Could this be null?

Reply:
No, because in `SessionSetupResponse` it is also not nullable